webpack.config.js

```diff
@@ -414,6 +414,14 @@ module.exports = {
                     }
                 }]
             },
+            {
+                test: /\.(eot|woff|woff2|svg|ttf)([\?]?.*)$/,
+                loader: 'file-loader',
```

Comment:
I still wonder about this bit. Are these just tiny fonts with one glyph or two for specialized widgets? But where would a table even use these? We really don't want to load complete fonts that way, we should instead serve them from cockpit/static in cockpit-ws, where they can be cached properly. What happens if you drop this?

Reply:
We don't seem to need it anymore. It previous versions of PF-react it webpack was dailing without this. Dropped it.